manifests/auth/base/auth-kube-applier.yaml

```diff
@@ -0,0 +1,24 @@
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: kube-applier-assumed
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kube-applier-sa-get
```

Comment:
This could be called just `kube-applier`?

Reply:
I quite like the fact that it isn't just `kube-applier` - there's too much 'kube-applier' over all the objects here, ha. 

What about `kube-applier-role`? Leaves it open to be appended to or amended.

Reply:
Its already a "role" - its like having a variable called "error_count_variable"

Reply:
There is a lot of `kube-applier` around, but that's common in kube ecosystem. If there aren't any other KA roles, I'd lend over it being called just `kube-applier`

Reply:
`kube-applier` it is